### ISSUE TITLE
SSH request for authenticate hidden

### ISSUE THREAD

**Issue by SijmenHuizenga** [tailscale/tailscale#7589]
### What is the issue?

When doing `tailscale ssh user@host`, the command is stuck and times out after some time. When adding `-v` to the command (`tailscale ssh user@host -v`), it appears that the ssh was stuck because of:

```
# Tailscale SSH requires an additional check.
# To authenticate, visit: https://login.tailscale.com/a/c2e8ee2fe1cf
```

Expected behavior is that the re-authentication request is always shown, even when no `-v` is added.

### Steps to reproduce

1. Do not use tailscale ssh for a day (or until re-authenticate is required)
2. Run `tailscale ssh user@host`
3. See that the command is stuck without output
4. Run `tailscale ssh user@host -v` to verify that the command is stuck because of an required re-authentication.

### OS

Linux

### OS version

Seen on both Arch and Ubuntu 20

### Tailscale version

Seen on both 1.36.2 and 1.38.1

### Other software

Tested with these two setups:

* Arch linux
* Tailscale 1.36.2
* OpenSSH_9.2p1, OpenSSL 3.0.8 7 Feb 2023

* Ubuntu 20
* Tailscale 1.38.1
* OpenSSH_8.2p1 Ubuntu-4ubuntu0.5, OpenSSL 1.1.1f  31 Mar 2020

### Bug report

BUG-07968cbbd1b7654b7ee1591ca95b7cb61ccc6120b703f0267b1ee10a596eb7ab-20230317094144Z-494dc391f87fdc7a

**Comment by DentonGentry** [comment]
One thing first: Tailscale SSH is implemented entirely on the server. One can use any SSH client to "ssh 100.x.y.z" and it will use Tailscale SSH.

Is there a reason to use `tailscale ssh` as the client?

**Comment by SijmenHuizenga** [comment]
> Is there a reason to use tailscale ssh as the client?

No, there is no reason. The behavior described in this issue shows up when using `tailscale ssh user@host` and `ssh user@host` as well.

**Comment by heliostatic** [comment]
This seems to be the issue I've got as well, in a slightly different manifestation: https://github.com/tailscale/tailscale/issues/7531

**Comment by DentonGentry** [comment]
Shortly before the bugreport we do see tailscaled receiving the check mode URL:

```
2023-03-17 09:38:23.427271934 +0000 UTC: Accept: TCP{100.104.x.y:22 > 100.101.x.y:53900} 60 tcp non-syn
2023-03-17 09:38:23.427492245 +0000 UTC: Accept: TCP{100.101.x.y:53900 > 100.104.x.y:22} 52 ok out
2023-03-17 09:38:23.73147702 +0000 UTC: control: netmap: control says to open URL https://login.tailscale.com/a/0123456789abcdef; opening...
```

**Comment by andrius** [comment]
Same here, were googling how to print this authentication message. Annoying. Of course I could create a script that can perform initial ssh check with -v but that's workaround

**Comment by SijmenHuizenga** [comment]
Found it! In this case the message was hidden because my ssh config was configured to only show `ERROR` level logs. The banner message from Tailscale is sent in `INFO` level, so it was hidden. After adding the following to `~/.ssh/config` file, the banner image shows up as expected:
```
LogLevel INFO
```
